tz/digest.go

```diff
@@ -122,3 +126,50 @@ func mulBitRightGeneric(c00, c10, c01, c11 *GF127, bit bool, tmp *GF127) {
 		*c11 = *tmp
 	}
 }
+
+// MarshalBinary implements [encoding.BinaryMarshaler].
+func (d *digest) MarshalBinary() ([]byte, error) {
+	var (
+		b          = make([]byte, Size)
+		start, end int
+	)
+
+	for i, a := range d.x {
+		state, err := a.MarshalBinary()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Not covered.

Reply:
GF127.MarshalBinary never returns an error